lib/teslamate/mqtt/pubsub/vehicle_subscriber.ex

```diff
@@ -39,7 +39,8 @@ defmodule TeslaMate.Mqtt.PubSub.VehicleSubscriber do
 
   @always_published ~w(charge_energy_added charger_actual_current charger_phases
                        charger_power charger_voltage scheduled_charging_start_time
-                       time_to_full_charge shift_state geofence trim_badging)a
+                       time_to_full_charge shift_state geofence trim_badging
+                       center_display_state)a
```

Comment:
You probably don't want or need it in the `always_published` list, this just means the value is published even the value is nil. We probably should have a comment to indicate what these list is used for.

The value does need to get included in the `simple_values` list though, below.